tests/integration/kubernetes/gha-run.sh

```diff
@@ -241,6 +241,66 @@ install_kata_tools_placeholder() {
 		"after CI picks up the gha yaml changes required to test that installation."
 }
 
+function update_worker_label(){
+	local workers
+	local label="$1"
+	workers="$(kubectl get nodes --no-headers | grep '\<worker\>' | awk '{ print $1 }')"
+	for nodename in $workers; do
+		if ! kubectl get "node/${nodename}" -o jsonpath='{.metadata.labels}' | \
+			grep "$label" >/dev/null; then
+			kubectl label node "$nodename" --overwrite "${label}"
+		fi
+	done
+}
+
+function deploy_snapshotter() {
+	echo "::group::Deploying ${SNAPSHOTTER}"
+
+	case ${SNAPSHOTTER} in
+		nydus) deploy_nydus_snapshotter ;;
+		*) >&2 echo "${SNAPSHOTTER} flavour is not supported"; exit 2 ;;
+	esac
+
+	echo "::endgroup::"
+}
+
+function deploy_nydus_snapshotter() {
+    echo "::group::deploy_nydus_snapshotter"
+
+	local nydus_snapshotter_install_dir="/tmp/nydus-snapshotter"
+	if [ -d "${nydus_snapshotter_install_dir}" ]; then
+		rm -rf "${nydus_snapshotter_install_dir}"
+	fi
+	mkdir -p "${nydus_snapshotter_install_dir}"
+	nydus_snapshotter_version=$(get_from_kata_deps "externals.nydus-snapshotter.version")
```

Comment:
I believe that here it is missing the code to clone the nydus_snapshotter project, right?

Another option is to wget/curl the nydus-snapshotter.yaml and nydus-snapshotter-rbac.yaml files only, edit them then apply.

Reply:
Fixed.